projects/go-lib/src/lib/components/go-icon-button/go-icon-button.component.scss

```diff
@@ -0,0 +1,18 @@
+@import '~@tangoe/gosheets/base/variables';
+@import '~@tangoe/gosheets/base/mixins';
+
+.go-icon-button {
+  @include transition(background-color);
+
+  border: 0;
+  border-radius: $global-radius--round;
+  color: $base-dark;
```

Comment:
Although they're the same color, we should probably use `$theme-light-color` for this.